08-numbers-and-strings/02-strings/exercises/08-right-trim-it/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf8"
+)
+
+func main() {
+	name := "inanç           "
+	name = strings.TrimRight(name, " ")
+	len := utf8.RuneCountInString(name)
```

Comment:
variable name `len`  collides with a built in function len
it will be better to rename it to `length` or `runeCount`